internal/identity/manager/manager.go

```diff
@@ -775,3 +779,21 @@ func isTemporaryError(err error) bool {
 	}
 	return false
 }
+
+func exponentialTry(ctx context.Context, f func() error) error {
```

Comment:
Maybe indicate this goes forever?

Reply:
it does not go forever. It goes until the function succeeds or context is Done.